_search-plugins/knn/knn-index.md

```diff
@@ -53,11 +53,7 @@ Starting with k-NN plugin version 2.19, you can use `binary` vectors with the `l
 
 ## SIMD optimization for the Faiss engine
 
-Starting with version 2.13, the k-NN plugin supports [Single Instruction Multiple Data (SIMD)](https://en.wikipedia.org/wiki/Single_instruction,_multiple_data) processing if the 
-underlying hardware supports SIMD instructions (AVX2 on x64 architecture and Neon on ARM64 architecture). SIMD is supported by default on Linux machines only for the Faiss engine. 
-SIMD architecture helps boost overall performance by improving indexing throughput and reducing search latency. Starting with version 2.18, the k-NN plugin supports AVX512 SIMD 
-instructions on x64 architecture and from version 2.19, the k-NN plugin supports advanced AVX512 SIMD instructions on x64 architecture if underlying system is an Intel® Sapphire 
-Rapids or a newer-generation processor which will boost the performance of hamming distance and fp16 computation.
+Starting with version 2.13, the k-NN plugin supports [Single Instruction Multiple Data (SIMD)](https://en.wikipedia.org/wiki/Single_instruction,_multiple_data) processing if the underlying hardware supports SIMD instructions (AVX2 on x64 architecture and Neon on ARM64 architecture). SIMD is supported by default on Linux machines only for the Faiss engine. SIMD architecture helps boost overall performance by improving indexing throughput and reducing search latency. Starting with version 2.18, the k-NN plugin supports AVX512 SIMD instructions on x64 architecture. Starting with version 2.19, the k-NN plugin supports advanced AVX512 SIMD instructions on x64 architecture for Intel Sapphire Rapids or a newer generation processor, improving the performance of Hamming distance and FP16 computation.
```

Comment:
Can you change the last line to `improving the performance of Hamming distance computation.` ?

As of now, looks like fp16 optimization is not yet added so let us remove it.